.github/workflows-source/schedule-daily.yml

```diff
@@ -230,3 +230,82 @@ jobs:
           compression-level: 9
           path: |
             cov_html.zip
+
+  recovery-engine-smoke-test:
+    name: Recovery Engine Smoke Test
+    <<: *dind-large-setup
+    timeout-minutes: 60
```

Comment:
the "smoke" in the test name suggests this is a very, very short test; is it expected to take up to 60mn?